parse/parse.go

```diff
@@ -0,0 +1,104 @@
+package utils
```

Comment:
The name of this package shouldn't be utils

Reply:
JSON parser maybe?

Reply:
only parser is good